README.md

```diff
@@ -1,57 +1,51 @@
 # bpmn-visualization 'getting started' tutorial
 
-A getting started demo that uses bpmn-visualization library to visualize runtime monitoring data.
+A getting started demo that uses bpmn-visualization library to visualize runtime monitoring data ⏲️.
 
 ## Content
+The following information is visualized:
+- Running instances and their number ⚙️
+- State of instances: whether or not they predefined KPIs ⏱️ are violated:
+    - Two KPIs are considered: completion and waiting time of activities.
+    - Two levels of violation are defined: risky 🟠 and criticial 🔴.
 
-The following visualization is used:
+## Visualization
 
-**Overlays on activities:**
+### Running instances and their number ⚙️
+- **animated activity stroke**: to indicate that there are running instances of the corresponding activitiy
 
-- green: number of instances running on time according to a predefined completion time KPI
-- orange: number of instances running late (with a risky level) according to a predefined completion time KPI
-- red: number of instances running late (with critical level) according to a predefined completion time KPI
+- **Overlays on activities**: to indicate the number of instances
 
-**Color of activities:**
+### State and KPI violation ⏱️
+**Colors meaning**:
+- 🟢 Green: no violation
+- 🟠 Orange: violation with a risky level
+- 🔴 Red: Violation with a critical level
 
-- green: avg execution time of instances is “efficient” according to a predefined KPI (throuhput time)
-- orange: avg execution time is “ineffcient with a risky level” according to a predefined KPI (throuhput time)
-- red: avg execution time is “ineffcient with a criticial level” according to a predefined KPI (throuhput time)
+#### Activities:
+- **Overlay filling color**: 
+    - Green: number of instances running on time according to a predefined completion time KPI

```

Comment:
ℹ️ you may add the emoji for colors as you did before

Reply:
fixed in bdebdea1f292fcaeec621bbf12e0f15e3a38aad3